requirements.txt

```diff
@@ -24,3 +24,6 @@ pymongo==3.1
 
 # Needed for the mysql_db module
 mysqlclient==1.3.7
+
+# pin cryptography to no newer than the newest release that hasn't dropped python2 support and added rust dependency
```

Comment:
btw I realized that the build process _is_ using python3, so clarifying here in case the comment above is misleading. :grimacing: The issue is the new rust dependency with the old version of pip used (missing setuptools-rust).